src/text/parsers/clob.rs

```diff
@@ -75,7 +82,7 @@ fn long_clob_body(input: &str) -> IResult<&str, Vec<u8>> {
 fn long_clob_fragment(input: &str) -> IResult<&str, StringFragment> {
     alt((
         escaped_newline,
-        escaped_char,
+        escaped_char_no_unicode,
```

Comment:
🗺️ Clobs aren't (necessarily) Unicode, so they don't recognize Unicode escape sequences. (`\u` and `\U`)